Decisions.TruCap/Steps/DocumentMonitorSteps.cs

```diff
@@ -27,21 +27,22 @@ public class DocumentMonitorSteps
             }
             
             string baseUrl = ModuleSettingsAccessor<TruCapSettings>.GetSettings().GetBaseDocumentMonitorUrl(overrideBaseUrl);
-            Task<HttpResponseMessage> response = TruCapRest.TruCapGet(
-                $"{baseUrl}/Project/{project}/DocumentSubType/{docSubType}/FromDate/{startDate}/ToDate/{endDate}",
-                authentication);
 
             try
             {
-                return DocumentMonitorResponse.JsonDeserialize(await response.Result.Content.ReadAsStringAsync());
+                Task<string> resultTask = TruCapRest.TruCapGet(
+                    $"{baseUrl}/Project/{project}/DocumentSubType/{docSubType}/FromDate/{startDate}/ToDate/{endDate}",
+                    authentication);
```

Comment:
If we're already waiting in `TruCapRest.TruCapGet` we should return the result and not the task.